mhs/mhs/common/state/tests/test_work_description.py

```diff
@@ -0,0 +1,157 @@
+import asyncio
+import json
+import unittest
+import mhs.common.state.work_description as wd
+from unittest.mock import MagicMock, patch
+from utilities.test_utilities import async_test
+
+DEFAULT_TABLE = 'default'
+
+input_data = {
+            wd.DATA_KEY: 'aaa-aaa-aaa',
+            wd.DATA: {
+                wd.TIMESTAMP: '12:00',
+                wd.VERSION_KEY: 1,
+                wd.STATUS: wd.MessageStatus.IN_OUTBOUND_WORKFLOW
+            }
+        }
+
+
+old_data = {
+            wd.DATA_KEY: 'aaa-aaa-aaa',
+            wd.DATA: {
+                wd.VERSION_KEY: 1,
+                wd.TIMESTAMP: '11:00',
+                wd.STATUS: wd.MessageStatus.IN_OUTBOUND_WORKFLOW
+            }
+        }
+
+
+class TestWorkDescription(unittest.TestCase):
```

Comment:
Looking at code coverage reports, it looks like tests are missing for:
- In `publish`, `latest_data is None`
- In `get_work_description_from_store`, `json_store_data is None`
- In `create_new_work_description`, `persistence_store  is None`

Reply:
Tried to parameterize the null checks in `create_new_work_description` but the parameterize library failed for `None` so I just had to make the test a bit longer